map/autoware_lanelet2_map_validator/docs/how_to_contribute.md

```diff
@@ -0,0 +1,120 @@
+# How to contribute to `autoware_lanelet2_map_validator`
+
+Your contribution is welcome to achieve a broad view of validation for lanelet2 maps.
+This document gives you the instructions on how to add a validator to `autoware_lanelet2_map_validator`.
+Please take a look at the [Design Concept](#design-concept) and follow the [Contribution Guide](#contribution-guide).
+
+## Design Concept
+
+The main goal of `autoware_lanelet2_map_validator` is to validate whether the lanelet2 map matches the vector map requirements for Autoware.
+`autoware_lanelet2_map_validator` achieves this by running a list of small validators.
+In other words, each vector map requirement will be validated by one or more validators.
+It is recommended to keep validators small and they don't have to be unique to a specific requirement so that we can broaden the expression of map requirements. (It doesn't mean that a validator should output only one kind of error!)
+
+The list of small validators will be defined as a JSON file (see [`autoware_requirement_set.json`](../autoware_requirement_set.json) for an example), and the output will also be a JSON file that appends validation results to a copy of the input. See [How to use `autoware_lanelet2_map_validator`](../README.md/#how-to-use) for further information about how the input and output are processed.
```

Comment:
> (see [`autoware_requirement_set.json`](../autoware_requirement_set.json)

Clicking on the links in the generated document redirects to the following URL:　https://autowarefoundation.github.io/autoware_tools/pr-170/map/autoware_lanelet2_map_validator/docs/autoware_requirement_set.json

Reply:
Links to other documents are also being redirected to `autoware_lanelet2_map_validator/docs/****`.

Reply:
Fixed by https://github.com/autowarefoundation/autoware_tools/pull/170/commits/7c20b390bbf511cd00a32685ebe58bc737d70bca https://github.com/autowarefoundation/autoware_tools/pull/170/commits/86634ca891dd75c3aa622521dface7c69a512af1 https://github.com/autowarefoundation/autoware_tools/pull/170/commits/17007b8bbfb48f4a686bef5a6f2dd57f73c85076